libraries/AP_NavEKF2/AP_NavEKF2_Control.cpp

```diff
@@ -519,6 +519,10 @@ void  NavEKF2_core::updateFilterStatus(void)
     filterStatus.flags.using_gps = ((imuSampleTime_ms - lastPosPassTime_ms) < 4000) && (PV_AidingMode == AID_ABSOLUTE);
     filterStatus.flags.gps_glitching = !gpsAccuracyGood && (PV_AidingMode == AID_ABSOLUTE) && !extNavUsedForPos; // GPS glitching is affecting navigation accuracy
     filterStatus.flags.gps_quality_good = gpsGoodToAlign;
+    // for reporting purposes we report rejecting airspeed after 3s of not fusing when we want to fuse the data
+    filterStatus.flags.rejecting_airspeed = lastTasFailTime_ms != 0 &&
+                                            (imuSampleTime_ms - lastTasFailTime_ms) < 1000 &&
+                                            (imuSampleTime_ms - lastTasPassTime_ms) > 3000;
```

Comment:
This condition will clear as soon as the first synthetic airspeed observation is fused, so we will get 1 second of filterStatus.flags.rejecting_airspeed == true. This does not seem long enough to ride out a stall or spin condition.

Reply:
testing in SITL shows it correctly setting rejecting_airspeed for the whole the airspeed is bad:
![image](https://user-images.githubusercontent.com/831867/151250320-fa6c565c-8aaa-40e2-8c65-1de636bcf2cf.png)
the test was done by setting ARSPD_RATIO=10 to give a silly high airspeed. We get rejecting_airspeed after 3s, and it persists until I set ARSPD_RATIO back to 2.0.

Reply:
To clarify, this would occur if bit 7 of FLIGHT_OPTIONS was true.

Reply:
I tested with FLIGHT_OPTIONS=128 and got the same result, the rejecting_airspeed flag is 1 for the whole time the airspeed is bad